### ISSUE TITLE
FR: padding & per-chart options

### ISSUE THREAD

**Issue by chrisgrieser** [alincoop/obsidian-tinychart#2]
First, thanks for this nice plugins. Just replaced the charts plugin with it.

So, two small annoyances I encountered are these:
1. The value labels are left-aligned. However, they should rather be right aligned:
```py
# current
1234
12

# desired
1234
  12
```

2. different charts have different label lengths, so that it would make sense for each chart to use individual length settings. However, it seems I can set the length only globally in the plugin settings, and not on a per-chart-basis.

Also, it would be nice to set the fill character on a per-chart basis as well to differentiate the charts.
![Pasted image 2024-05-03 at 20 47 35@2x](https://github.com/alincoop/obsidian-tinychart/assets/73286100/9f27417a-dd24-4b8d-9a0a-b0994b042f43)

**Comment by alincoop** [comment]
Thank you, Chris.

I have added the option to right-align the value labels (see version 0.1.1) and will work on per-chart configuration options, to override global settings such as chart length.

**Comment by alincoop** [comment]
@chrisgrieser - Regarding chart length, what do you think of the following approaches which standardise the total chart length by either padding the label (A) or expanding the bar chart (B)?

Let me know if this would be helpful and which style you prefer, I can implement it sooner than per-chart settings.

<img width="762" alt="image" src="https://github.com/alincoop/obsidian-tinychart/assets/17536899/e3aabfa9-a872-4b7b-a379-0db8c6546915">

**Comment by chrisgrieser** [comment]
> what do you think of the following approaches which standardise the total chart length by either padding the label (A) or expanding the bar chart (B)?

yes, I think this would be a good solution. Saves you the trouble of having to set the length for each graph individually
